lib/source-code/source-code.js

```diff
@@ -410,6 +420,28 @@ class SourceCode extends TokenStore {
         return result;
     }
 
+    /**
+     * Returns text between two tokens.
+     * @param {Token} first The token to get text after.
+     * @param {Token} second The token to get text before.
+     * @returns {string} Text between specified tokens.
+     * @public
+     */
+    getTextBetweenTokens(first, second) {
```

Comment:
I'm not opposed to adding this to our public API, but I think the other two added methods should be implemented in rules or, if shared, in `ast-utils`.

Reply:
These methods may be useful for other rules too.
I'm not sure about moving these methods into `ast-utils`.
I've extracted code for methods from `isSpaceBetweenTokens` and made the code reusable.
If considering this idea, then `isSpaceBetweenTokens` can also be moved into `ast-utils`.